File: src/game/events/game/AllEnemiesEliminatedGameEvent.java
package game.events.game;

import game.Bomberman;
import game.events.models.GameEvent;

public class AllEnemiesEliminatedGameEvent implements GameEvent {
    @Override
    public void invoke(Object arg) {
        Bomberman.getMatch().getCurrentLevel().onAllEnemiesEliminated();
    }
}

File: src/game/events/game/AllEnemiesEliminatedGameEvent.kt
package game.events.game

import game.Bomberman
import game.events.models.GameEvent

class AllEnemiesEliminatedGameEvent : GameEvent {
    override fun invoke(arg: Any?) {
        Bomberman.getMatch().currentLevel!!.eventHandler.onAllEnemiesEliminated()
    }
}